.github/workflows/fake-bpy-module-ci.yml

```diff
@@ -134,6 +134,9 @@ jobs:
           name: fake_bpy_modules_${{ matrix.blender_version}}_raw_${{ needs.set_versions.outputs.file_version }}
           path: "raw_modules/fake_bpy_module*"
 
+      - name: Run Ruff flake8-pyi checks
```

Comment:
The generated code can be found at the path `raw_modules/fake_bpy_module*` as zip files.
Extract them to the some temporary directories and lint them is better which tests the artifacts itself.